services/collaboration/pkg/internal/app/app.go

```diff
@@ -0,0 +1,111 @@
+package app
+
+import (
+	"context"
+	"errors"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
+	"github.com/owncloud/ocis/v2/services/collaboration/pkg/config"
+
+	registryv1beta1 "github.com/cs3org/go-cs3apis/cs3/app/registry/v1beta1"
+	gatewayv1beta1 "github.com/cs3org/go-cs3apis/cs3/gateway/v1beta1"
+	rpcv1beta1 "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	"github.com/cs3org/reva/v2/pkg/mime"
+	"github.com/cs3org/reva/v2/pkg/rgrpc/todo/pool"
+	"github.com/gofrs/uuid"
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/envdecode"
+	"github.com/owncloud/ocis/v2/ocis-pkg/registry"
+	"google.golang.org/grpc"
+)
+
+type DemoApp struct {
```

Comment:
no demo - right?

Reply:
internal package needs a heavy refactor because everything is there. The "DemoApp" should be either removed or replaced by something else.